lib/active_campaign_crm/client/tags.rb

```diff
@@ -0,0 +1,31 @@
+module ActiveCampaignCrm
+  class Client
+    module Tags
+      def tags(params = {})
+        response = @connection.get('tags', params)
+        response['tags']
+      end
+
+      def tag(id)
+        response = @connection.get("tags/#{id}")
+        response['tag']
+      end
+
+      def create_tag(fields)
+        @connection.post('tags', tag_body(fields))
+      end
+
+      def update_tag(id, fields)
+        @connection.put("tags/#{id}", tag_body(fields))
+      end
+
+      def delete_tag(id)
+        @connection.delete("tags/#{id}")
+      end
+
+      def tag_body(fields)
+        { 'tag': fields }.to_json
```

Comment:
Which kind of data will be in fields?

Reply:
properties of the tag, like name, description and type